### ISSUE TITLE
How should a model be deconstructed

### ISSUE THREAD

**Issue by stigmergic** [backspaces/as-app3d#35]
We are creating, destroying and then creating new models.

After doing this enough times we start seeing the following warning:

`WARNING: Too many active WebGL contexts. Oldest context will be lost.`

This is likely because we are not properly removing the model from the page and cleaning up ThreeJS.

**Comment by backspaces** [comment]
Yup. Emma ran into that and found a way to clean it up. I think. :)

**Comment by backspaces** [comment]
gl.getExtension('WEBGL_lose_context').loseContext();

**Comment by backspaces** [comment]
BTW, in the old util.imageToBytes utility for letting us use RGBA for DataSets (i.e. the alpha channel too), we used a webgl context and ran into the same problem. Then we refactored the context out of the function to be a util.imageToBytesCtx property, building it only once .. reusing it later.

Emma will know how Three handles this and I can put it into the ThreeView.

**Comment by stigmergic** [comment]
@egouldmarie thoughts?
